### PR TITLE
use Cascade to set dynannotate ViewTemplate

### PR DESCRIPTION
**Pull request by oflg**
It will prevent dynannotate ViewTemplate from affecting other normal ViewTemplates.

### CODE REVIEW

File: plugins/tiddlywiki/dynannotate/examples/viewtemplate.tid [hunk 1]
```diff
@@ -16,13 +16,12 @@ Once enabled, this example demonstrates several features:
...
 <$button>
-<$action-setfield $tiddler="$:/core/ui/ViewTemplate/body" $field="text" $value={{$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text}}/>
-<$action-setfield $tiddler="$:/core/ui/ViewTemplate/body" $field="tags" $value="$:/tags/ViewTemplate"/>
+<$action-setfield $tiddler="$:/config/ViewTemplateTitleFilters/fish" tags="$:/tags/ViewTemplateTitleFilter" text="[[$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text]]" list-before="$:/config/ViewTemplateBodyFilters/default"/>
 Click here to use the custom view template
 </$button>
 
 <$button>
-<$action-deletetiddler $tiddler="$:/core/ui/ViewTemplate/body"/>
+<$action-deletetiddler $tiddler="$:/config/ViewTemplateTitleFilters/fish"/>
```
Comment (pmario) [inline plugins/tiddlywiki/dynannotate/examples/viewtemplate.tid:24]: Where does this config come from? IMO it shouldn't be there at all.

Comment (oflg) [reply plugins/tiddlywiki/dynannotate/examples/viewtemplate.tid:24]: Sorry. I forgot to change the name in the test case. I will correct it.

Comment (oflg) [reply plugins/tiddlywiki/dynannotate/examples/viewtemplate.tid]: > IMO it shouldn't be there at all.

I have fixed it to: `$:/config/ViewTemplateTitleFilters/dynannotate`

**Comment by Jermolene** [comment]
Great stuff, thank you @oflg. I don't think you've signed the CLA. With apologies for the bureaucracy, please could you prepare a separate PR to the 'tiddlywiki-com' branch with your signature for the Contributor License Agreement (see [contributing.md](https://github.com/Jermolene/TiddlyWiki5/blob/master/contributing.md)).

**Comment by oflg** [comment]
> don't think you've signed the CLA.

I should have read the contribution readme carefully and signed CLA.😥

I am signing it now: https://github.com/Jermolene/TiddlyWiki5/pull/6526

**Comment by Jermolene** [comment]
Thanks @oflg that's great.

**Pull request merged**